pyproject.toml

```diff
@@ -16,7 +16,7 @@ authors = [
 ]
 description = "Shapash is a Python library which aims to make machine learning interpretable and understandable by everyone."
 readme = "README.md"
-requires-python = ">3.8, <3.13"
+requires-python = ">3.8, <3.14"
```

Comment:
I think it might be better to change this to:

`requires-python = ">=3.9, <3,14"`

See [this comment](https://github.com/MAIF/shapash/pull/634#issuecomment-2814722367) for the rationale.